platform/server-bundle/src/main/resources/webapp/js/services.js

```diff
@@ -87,6 +87,8 @@
                 dialog.getModalBody().css('padding', '0');
 
                 dialog.open();
+
+                dialog.getModal().prev().css('z-index', dialog.getModal().css('z-index'));
```

Comment:
@sebbrudzinski This is hack, indeed, but there's no other way to fix it. Problem appears when we call more than 1 modal on page. Every modal is preceded by div which is covering whole page - always. Bootstrap is setting too low z-index to this div. Without manual fix of z-index the new modal will be always on the same z-index with previous modal.